go.mod

```diff
@@ -1,9 +1,10 @@
 module github.com/terraform-redhat/terraform-provider-rhcs
 
-go 1.18
+go 1.20
```

Comment:
Can you run 'go mod tidy' and commit the resulted changes?
it should fix the CI

Reply:
that's what I'm trying to do
seems to not work :/

Reply:
fixed by changing `check-pull-request.yaml` `go-version` to 1.20